src/components/Invoices/InvoicesTable.tsx

```diff
@@ -198,8 +258,62 @@ export const InvoicesTable = () => {
     }
   }, [fetchMore, hasMore])
 
+  const options: InvoiceStatusOption[] = useMemo(() => Object.values(InvoiceStatusOptionConfig), [])
+
+  const SelectedValue = useMemo(() => {
+    const label = selectedInvoiceStatusOption?.label
+    return label ? `Status: ${label}` : 'Status'
+  }, [selectedInvoiceStatusOption?.label])
+
+  const StatusFilter = useCallback(() => (
+    <ComboBox
+      className='Layer__InvoicesTable__StatusFilter'
+      options={options}
+      onSelectedValueChange={option => setSelectedInvoiceStatusOption(option)}
+      selectedValue={selectedInvoiceStatusOption}
+      isSearchable={false}
+      isClearable={false}
+      placeholder='Status'
+      slots={{ SelectedValue }}
+    />
+  ),
+  [SelectedValue, options, selectedInvoiceStatusOption])
+
+  const CreateInvoiceButton = useCallback(() => (
+    <Button>
+      Create Invoice
+      <Plus size={16} />
+    </Button>
+  ),
+  [])
+
+  const InvoicesTableEmptyState = useCallback(() => {
+    const isFiltered = selectedInvoiceStatusOption && selectedInvoiceStatusOption !== ALL_OPTION
+
+    return (
+      <DataState
+        status={DataStateStatus.allDone}
+        title={isFiltered ? 'No results found' : 'No invoices yet'}
+        description={
+          isFiltered
+            ? 'We couldn’t find any invoices with the current filters. Try changing or clearing them to see more results.'
+            : 'Add your first invoice to start tracking what your customers owe you.'
+        }
+        icon={isFiltered ? <Search /> : <HandCoins />}
+        spacing
+      />
+    )
+  }, [selectedInvoiceStatusOption])
+
   return (
-    <Container name='InvoiceTable'>
+    <Container name='InvoicesTable'>
+      <DataTableFilters
```

Comment:
I don't actually see the `DataTableFilters` component committed - am I doing something silly?